gateway/main/settings.py

```diff
@@ -89,17 +92,40 @@
 
 WSGI_APPLICATION = "main.wsgi.application"
 
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+        },
+    },
+    "root": {
+        "handlers": ["console"],
+        "level": "DEBUG",
```

Comment:
We can open an issue and do it later but we should be able to modify the level of the LOGs through an environment variable, for example.